setup.cfg

```diff
@@ -3,10 +3,12 @@ universal = 1
 
 [flake8]
 ignore =
+    # E203: Whitespace before ':'; doesn't work with black
+    E203,
     # E501: line too long
     E501,
-    # W504: line break after binary operator
-    W504,
+    # W503: line break before operator; this doesn't work with black
+    W503,
```

Comment:
Previously, we did line breaks after operators. Black does this differently, thus the switch.